src/main/java/org/generationcp/middleware/service/impl/feedback/FeedbackServiceImpl.java

```diff
@@ -0,0 +1,99 @@
+package org.generationcp.middleware.service.impl.feedback;
+
+import java.util.Optional;
+
+import org.generationcp.middleware.ContextHolder;
+import org.generationcp.middleware.hibernate.HibernateSessionProvider;
+import org.generationcp.middleware.manager.WorkbenchDaoFactory;
+import org.generationcp.middleware.pojos.workbench.WorkbenchUser;
+import org.generationcp.middleware.pojos.workbench.feedback.Feedback;
+import org.generationcp.middleware.pojos.workbench.feedback.FeedbackFeature;
+import org.generationcp.middleware.pojos.workbench.feedback.FeedbackUser;
+import org.generationcp.middleware.service.api.feedback.FeedbackService;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@Transactional
+public class FeedbackServiceImpl implements FeedbackService {
+
+	@Value("#{new Boolean('${feedback.enabled}')}")
+	private boolean feedbackEnabled;
```

Comment:
to better understand the code I think we can rename the attribute to feedbackFeatureEnabled 
@lik-83

Reply:
Done